ios/FluentUI/Table View/TableViewHeaderFooterView.swift

```diff
@@ -163,10 +163,10 @@ open class TableViewHeaderFooterView: UITableViewHeaderFooterView {
     @objc open var onAccessoryButtonTapped: (() -> Void)?
     @objc open var onHeaderViewTapped: (() -> Void)?
 
-    /// configure this variable to change the appropriate background color based on what type of UITableView style it is in hosted
-    @objc public var tableViewStyle: UITableView.Style = .insetGrouped {
+    /// configure this variable to change the appropriate background color based on what type of TableViewCell style it is associated with
+    @objc public var tableViewCellStyle: TableViewCellBackgroundStyleType = .grouped {
```

Comment:
Breaking change - this will have to be fixed everywhere in devmain. Is this a necessary change?

Reply:
i justed added that api this morning so no breaking change

Reply:
The tableViewStyle -> tableViewCellStyle rename isn't new? Should this be rebased off a different change then?

Reply:
talked offline. https://github.com/microsoft/fluentui-apple/pull/1570